installer/manifests/keptn/templates/secret-service.yaml

```diff
@@ -48,6 +48,10 @@ spec:
       {{- include "keptn.common.pod-security-context" . | nindent 6 }}
       serviceAccountName: keptn-secret-service
       terminationGracePeriodSeconds: {{ .Values.secretService.gracePeriod | default 60 }}
+      initContainers:
+        {{- include "keptn.initContainers.wait-for-nats" . | nindent 8 }}
+        {{- include "keptn.initContainers.wait-for-mongodb-datastore" . | nindent 8 }}
+        {{- include "keptn.initContainers.wait-for-shipyard-controller" . | nindent 8 }}
```

Comment:
Also the secret-service has not dependency

Reply:
removed